core/src/scss/core/helpers/_aspect-ratio.scss

```
@@ -16,10 +16,10 @@
   @include aspect-ratio(16, 9);
 }
 
-.su-aspect-ratio-4x3 {
+.su-aspect-ratio--4x3 {
```

```suggestion
.su-aspect-ratio-4x3, // Deprecated (su-aspect-ratio-4x3) to be removed in V6.
.su-aspect-ratio--4x3 {
```

For backwards compatibility with v5 keep the original and add the change with a deprecation notice.